### DISCUSSION TITLE
How do I use the license key?

### DISCUSSION THREAD

**Discussion by L3oJr** [PrismLibrary/Prism#3262] [.NET MAUI]
Hi, the company where I'm working bought the commercial license and activated it on my email. I received the license key, but how do I use it? Do I need to insert the license key somewhere in the code?

Thanks.

**Comment by brianlagunas** [comment]
Currently, a standard Commercial license key is not used. Only a Commercial Plus license key is used to access the Plus benefits. However, this will be changing in a future release.

**Reply by a-einstein** [reply]
I don't understand.
After updating packages on Windows WPF, I have build errors
In the ReadMe there is a story about licenses.
I registered a Community License.
I found a License Key.
Now what?
Is this all for nothing? Is it unrelated?

**Reply by brianlagunas** [reply]
It's unrelated.

**Reply by dansiegel** [reply]
> I don't understand. After updating packages on Windows WPF, I have build errors In the ReadMe there is a story about licenses. I registered a Community License. I found a License Key. Now what? Is this all for nothing? Is it unrelated?

Slightly longer answer @brianlagunas is nicer than I am... I wanted to break your build if you didn't include a license as part of your build... he didn't at least for 9.0... so whatever your issue is, is unrelated to the updated license requirements.
